libs/surfaces/midi_surface/wscript

```diff
@@ -10,7 +10,8 @@ def options(opt):
     autowaf.set_options(opt)
 
 def configure(conf):
-    pass
+    autowaf.check_pkg(conf, 'glib-2.0', uselib_store='GLIB', atleast_version='2.73', mandatory=True)
+    autowaf.check_pkg(conf, 'glibmm-2.4', uselib_store='GLIBMM', atleast_version='2.32.0', mandatory=True)
```

Comment:
These are already checked in the top level wscript and should thus not be necessary here. Duplicate and slightly conflicting definitions of the uselib flags could potentially be confusing. But I hope to clean that up after other wscript cleanup has landed.

Reply:
Ok, I didn't see this. I can clean this up, too.